src/devices/Rfid/Data212_424kbps.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Rfid
+{
+    /// <summary>
+    /// 212 and 424 card elements
+    /// </summary>
+    public class Data212_424kbps
```

Comment:
(optional for first iteration) why is kbps in the name? the struct doesn't seem to be related with any speed. Is there any better name than number for this perhaps?

Reply:
The data returned are different depending on the speed communication between the card and the module. so kbps is actually defining what the number are. I would keep it for clarity